ui/doc/developer-setup.md

```diff
@@ -0,0 +1,35 @@
+# Developer Setup
+
+## Prequisties
+
+- Linux or MacOS
+- [VS Code](https://code.visualstudio.com/download)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
```

Comment:
We should mention `Python` here as well, as it will be needed for running the `ui/run.py` helper.